frontend/src/Components/Management/AccountList.tsx

```
@@ -1,28 +1,34 @@
-import React from 'react'
-import ItemCardHeader from './ItemCardHeader'
-import { GetAccountByPageResponse } from '@/Interfaces/Account/Schema/account'
+'use client'
+
+import React, { useState } from 'react'
 import ItemCard from './ItemCard'
+import { ITEMS_PER_PAGE_COUNT } from '@/Constants/Management'
+import { useGetAccountsByPage } from '@/Services/account-service'
+import Pagination from './Pagination'
+
+const AccountList = ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)